rebasehelper/tags.py

```diff
@@ -22,29 +22,33 @@
 #          Nikola Forró <nforro@redhat.com>
 #          František Nečas <fifinecas@seznam.cz>
 
+import collections
 import collections.abc
 import fnmatch
 import re
-from typing import Iterator, Optional, Tuple, cast
+from typing import Dict, Iterator, List, Optional, Tuple, cast
 
 from rebasehelper.spec_content import SpecContent
 from rebasehelper.helpers.macro_helper import MacroHelper
 
 
 class Tag:
-    def __init__(self, section: str, line: int, name: str, value_span: Tuple[int, int], valid: bool) -> None:
+    def __init__(self, section: str, line: int, name: str, value_span: Tuple[int, int], valid: bool,
+                 section_index: int) -> None:
         self.section: str = section
```

Comment:
I know it's backwards incompatible change, but I would prefer `section_name` here for clarity. Also I think `section_index` should be positioned above it, because it's the canonical reference.

Reply:
Changing the order definitely makes sense. But do we really want to make a backwards incompatible change? The `section` attribute is used several times. I don't think that the difference is that big to be worth renaming the attribute.

Reply:
> The `section` attribute is used several times.

What do you mean by that?

Reply:
For example: https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/specfile.py#L207

There are some more uses of the attribute in `ReplaceOldVersionHook`

Reply:
Well of course it is used, otherwise it wouldn't be there. But that doesn't mean it makes sense not to rename it. Your examples need to be rewritten anyway, they wouldn't work with non-unique section names.